admin/users/views.py

```diff
@@ -1,17 +1,35 @@
+from django.shortcuts import redirect
+from django.views.defaults import page_not_found
+
 from website.project.model import User
 
-from admin.base.views import GuidFormView
+from admin.base.views import GuidFormView, GuidView
 from admin.users.templatetags.user_extras import reverse_user
 from .serializers import serialize_user
 
 
+def remove_2_factor(request, guid):
+    user = User.load(guid)
+    try:
+        user.delete_addon('twofactor')
+    except AttributeError:
```

Comment:
This is rather indirect. Rather than handling an `AttributeError` here, you can check if `user` is `None`.

```python
if user:
   page_not_found(request)
```

Since this a very common operation, you might want to write a helper along the lines of [get_object_or_error](https://github.com/CenterForOpenScience/osf.io/blob/219c0fce2123489ae21abbac4a7f0d2b4b583985/api/base/utils.py#L56), which allows you to do

```python
user = get_object_or_error(User, guid)
user.delete_addon('twofactor')
```